app/models/childrens_barred_list_entry.rb

```diff
@@ -1,5 +1,9 @@
 class ChildrensBarredListEntry < ApplicationRecord
   validates :first_names, presence: true
-  validates :last_name, presence: true
+  validates :last_name,
+            presence: true,
+            uniqueness: {
+              scope: %i[first_names date_of_birth]
```

Comment:
This made me think - if there were two records with same `last_name`, `first_names` and `date_of_birth`, but one of them _also_ had a `trn` / `national_insurance_number`, we might want to keep the one with more info / compare the two.

If that ever happened we should probably handle that manually though, rather than in code.

Reply:
Ah yes, if the first record to make it to the db is missing the TRN we'd never update it. I will make sure we update existing matches.